lib/src/kbasesearchengine/parse/ObjectParser.java

```diff
@@ -80,7 +121,8 @@ public static String extractParentFragment(ObjectTypeParsingRules parsingRules,
             if (!rules.isFromParent()) {
                 continue;
             }
-            //TODO CODE this seems wrong. Why adding null paths?
+            // This list add is simply a check to see if at least one rule for
+            // extracting a parent element exists.
```

Comment:
I'm not sure this is true - the indexing paths are used in line 92/134 below.

Reply:
Yep. Removed comment.